packages/circularprogress/package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@pluralsight/ps-design-system-core": "^4.3.4",
     "@pluralsight/ps-design-system-theme": "^1.3.3",
-    "polished": "^1.7.0",
+    "polished": "^1.9.2",
```

Comment:
just an errant thought:  this is pretty large for what we're using it for (I believe, just transparentize, ever).  Would be cool to maybe hork that function and drop this package someday (not this PR).

Reply:
i don't know much about it - i'm curious to see what else it does